flutter/lib/desktop/pages/connection_page.dart

```diff
@@ -142,8 +147,68 @@ class _ConnectionPageState extends State<ConnectionPage>
     connect(context, id, isFileTransfer: isFileTransfer);
   }
 
+  Future<void> _fetchPeers() async {
+    setState(() {
+      isPeersLoading = true;
+    });
+    await Future.delayed(Duration(milliseconds: 100));
+    await _getAllPeers();
+    setState(() {
+        isPeersLoading = false;
+      });
+  }
+
+  Future<void> _getAllPeers() async {
+    Map<String, dynamic> recentPeers = jsonDecode(await bind.mainLoadRecentPeersSync());
+    Map<String, dynamic> favPeers = jsonDecode(await bind.mainLoadFavPeersSync());
```

Comment:
No need to load fav, because fav items must be in the other tabs, please confirm.

Reply:
yes, I will remove fav.